pkgs/development/python-modules/textnets/default.nix

```diff
@@ -63,5 +63,6 @@ buildPythonPackage rec {
     changelog = "https://github.com/jboynyc/textnets/blob/v${version}/HISTORY.rst";
     license = licenses.gpl3Only;
     maintainers = with maintainers; [ jboy ];
+    broken = lib.versionAtLeast pandas.version "2";
```

Comment:
Upstream says that there is support for panda > 2.0 for awhile ([0.9.0](https://github.com/jboynyc/textnets/blob/v0.9.0/pyproject.toml#L27)).

Reply:
Indeed ! I updated the package instead (however it doesn't build because of broken `tensorflow` dependency.